### ISSUE TITLE
License

### ISSUE THREAD

**Issue by ak9250** [iPERDance/iPERCore#14]
Hi, can you please add a License file thanks

**Comment by ak9250** [comment]
@piaozhx @StevenLiuWen any update on this?

**Comment by StevenLiuWen** [comment]
Hi, @ak9250 ,
Sorry for the late reply. Thanks for your attention to our work, currently we are preparing the LICENSE of the impersonator++ project. 
The whole project is Apache License 2, http://www.apache.org/licenses/LICENSE-2.0. This project is available for commercial usage, and you only need to cite our codes if you have used the code from the impersonator++ project.

However, this project relies on some third packages, and you need also obey the license of the third package, including:
	Pytorch: available for commercial usage;
	MMCV: Apache License 2, available for commercial usage;
	MMEditing: Apache License 2, available for commercial usage;
	MMDetection: Apache License 2, available for commercial usage;

	OpenPose: the original OpenPose is non-commercial and is built on Caffe framework. In the impersonator++ project, we convert it to PyTorch codes, but the pre-trained model comes from the original Caffe model. We convert the Caffe model (http://posefs1.perception.cs.cmu.edu/OpenPose/models/pose/body_25/pose_iter_584000.caffemodel)  to the Pytorch model (./assets/checkpoints/pose2d/ openpose_body25.pth).
	SMPL: The most important one is the SMPL (with its model), which is very restricted for commercial usage.

In summary, the codes from impersonator++ are available for commercial usage (cite us). 
The third packages, Pytorch, MMCV, MMEditing, and MMDetection, are all available for commercial usage.
However, it would be best to get the license grant for the commercial usage of OpenPose pre-trained and SMPL pre-trained models (see https://smpl.is.tue.mpg.de/modellicense for more details).

We will update our LICENSE later on Github.
